xls/dslx/bytecode/bytecode_interpreter_test.cc

```diff
@@ -1519,6 +1519,78 @@ fn main() {
   EXPECT_TRUE(value.IsUnit());
 }
 
+// GH 981
+// Assert_eq type deduction didn't fail in
```

Comment:
not sure if we need this extract docstring if we rename the test.

Reply:
Doc string moved closer to `send_if`.